app/views/custom_attributes/custom_attribute_definitions/_form_fields_only.html.erb

```diff
@@ -10,8 +10,19 @@
 
 <div class="custom-attribute-definition title <%=titleClass%>">Type:</div>
 <div class="custom-attribute-definition field <%=fieldClass%>">
-  <%= f.select :attr_type, custom_attribute_types.except!(CustomAttributes::CustomAttribute::TYPE_MULTIPLE_CHOICE, CustomAttributes::CustomAttribute::TYPE_CHECKBOX).invert, {}, { disabled: !new_record, id: "custom_attribute_type" } %>
+  <% custom_attr_type_hash = custom_attribute_types.except!(CustomAttributes::CustomAttribute::TYPE_MULTIPLE_CHOICE, CustomAttributes::CustomAttribute::TYPE_CHECKBOX).invert %>
+  <% type_options = { disabled: !new_record, id: "custom_attribute_type" } %>
+
+  <% 
+    if local_assigns[:is_date_alert]
```

Comment:
why flag passed and checked why not the type check of current attribute like `TYPE_DATE_TIME`